### ISSUE TITLE
license unclear

### ISSUE THREAD

**Issue by maltegrosse** [NREL/eclipse-2017#1]
Hello, the license (LICENSE.md) does not fit to the different libraries from 
https://github.com/NREL/eclipse-2017/tree/master/pv-generation-profiles/generation-profiles/lib/SAMPA

**Comment by dhetting** [comment]
No, we can’t change the license. Is there a specific use case that isn’t clear?
